docs-website/pages/plugins-actions.md

```diff
@@ -1,8 +1,14 @@
 ---
-title: Action API
+title: Actions
 summary: API to let plugins react to events in the app.
-permalink: /guides/action-api/
-order: 140
+section: plugins
+chapter: JavaScript API
+permalink: /plugins/actions
+
+redirect_from:
+  - /guides/action-api/
+
+order: 304
 ---
 
 In Sketch 3.8, we introduced the Action API: a way to let plugins react to events in the app. Using it, plugin authors can write code that is executed when certain actions are triggered, like "opening a document", "saving", "adding an artboard"…
```

Comment:
On this page it might make sense to communicate that the dev tools provide a really easy way to listen for all actions.

![Screen Shot 2019-04-20 at 12 20 29 AM](https://user-images.githubusercontent.com/4199296/56454207-21395280-6302-11e9-97d9-e5e3f2930062.png)